packages/cli/src/commands/env/pull.ts

```diff
@@ -100,6 +110,7 @@ export default async function pull(
   const records = (
     await pullEnvRecords(output, client, project.id, source, {
       target: environment || ProjectEnvTarget.Development,
+      gitBranch,
```

Comment:
Was the API updated to merge env vars? Previously we needed two API calls:

1. fetch all preview env vars
2. fetch preview env vars for the git branch

Then the client had to merge.

Does the API do the merging now? It might be good to add a test.

Reply:
Yes it merges.